ros2_batch_job/util.py

```diff
@@ -183,6 +183,45 @@ def on_process_exited(self, returncode):
                 fargs=(" ".join(self.cmd), returncode))
 
 
+class PipProtocol(AsyncSubprocessProtocol):
+    def __init__(self, cmd, exit_on_error, *args, **kwargs):
+        self.cmd = cmd
+        self.exit_on_error = exit_on_error
+        self.progress_bar = False
+        self.progress_data = ''
+        AsyncSubprocessProtocol.__init__(self, *args, **kwargs)
+
+    def on_stdout_received(self, data):
+        # The 'pip install' progress bar output can be pretty noisy in CI output.  Print only every 10th
```

Comment:
This comment is now slightly wrong.  Now we print only once we've gotten 1000 characters into the string, and when we do print, we just print the last line that came in.  So please update the comment to reflect that.